src/modules/FileActionsMenu/FileActionsMenu/resource.base.h

```diff
@@ -0,0 +1,13 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
```

Comment:
no need for .base.h file if there are no strings to localize in this dll, i.e. if not running convert-resx-to-rc.ps1